.eslintrc

```diff
@@ -38,9 +37,8 @@
         "react/no-unknown-property": 1,
         "react/prop-types": 1,
         "react/react-in-jsx-scope": 1,
-        "react/require-extension": 1,
```

Comment:
Unlike the other eslint rules here that were simply renamed, this one was changed to a different plugin ([see here](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md)). It would be great to add that plugin and configure some of its other rules as well, but I thought that might be more appropriate for a different pull request.